### PR TITLE
Fixes bug where images don't show up in templates and boards created from templates

### LINKED ISSUE #2223: Bug: images don't show up in templates created from boards
## Steps to reproduce the behaviour

1. Create a new board.
2. Add some cards with images in the card's description.
3. Create a template from this board.
4. Edit the template and open the card in which you added the images. Notice the images don't load up and you get network errors in the browser console.
5. Create a board from this template and open the card in which you added the images. Notice the images don't load up and you get network errors in the browser console.

## Expected behaviour

Card images should show up in templates and boards created from templates.

## Screenshots

[If applicable, add screenshots to elaborate on the problem.](https://user-images.githubusercontent.com/18575143/152153086-07fd7cef-157e-4989-8230-03142725235a.mp4)

## Edition and Platform

 - Edition: Mattermost plugin
 - Version: latest `main`
 - Browser and OS: platform-independent bug

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Fixes bug where images don't show up in templates and boards created from templates

#### Ticket Link
Fixes #2223

### CODE REVIEW

File: server/app/blocks.go [hunk 1]
```diff
@@ -133,6 +133,36 @@ func (a *App) InsertBlocks(c store.Container, blocks []model.Block, modifiedByID
 	return blocks, nil
 }
 
+func (a *App) UpdateFileIDs(sourceBoardID string, blocks []model.Block) error {
+	// Images attached in cards have a path comprising the card's board ID.
+	// When we create a template from this board, we need to copy the files
+	// with the new boar ID in path.
```
Comment (wiggin77) [inline server/app/blocks.go:139]:
```suggestion
	// with the new board ID in path.
```

**Comment by ogi-m** [comment]
Thanks @harshilsharma63 , LGTM

File: server/app/blocks.go [hunk 2]
```diff
@@ -133,6 +133,36 @@ func (a *App) InsertBlocks(c store.Container, blocks []model.Block, modifiedByID
 	return blocks, nil
 }
 
+func (a *App) UpdateFileIDs(sourceBoardID string, blocks []model.Block) error {
```
Comment (wiggin77) [inline server/app/blocks.go:136]: This isn't actually updating the fileIDs, it's copying files.  Perhaps a different name would be clearer.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:   Perhaps rename the app layer method to convey it is copying files.  Great work!

**Review by sbishel — APPROVED** [review]

**Pull request merged**